src/main/java/ibas/inchelin/web/dto/notice/NoticeContentResponse.java

```diff
@@ -1,3 +1,17 @@
 package ibas.inchelin.web.dto.notice;
 
-public record NoticeContentResponse(String content) {}
\ No newline at end of file
+import ibas.inchelin.domain.notice.entity.Notice;
+import lombok.Getter;
+
+public class NoticeContentResponse {
+
+    //공지 내용 조회를 위한 dto
+    @Getter
+    public static class NoticeDetailDto {
+        private String content;
+
+        public NoticeDetailDto(Notice notice) {
+            this.content = notice.getContent();
+        }
+    }
+}
```

Comment:
유지보수성을 위해 코드 스타일을 통일하고자 합니다.
따라서, 기존에 만들어둔 DTO는 수정 없이 그대로 사용해 주시면 좋겠습니다.
```suggestion
public record NoticeContentResponse(String content) {}
```